packages/editor-ui/src/n8n-theme.scss

```diff
@@ -213,3 +213,7 @@ body {
 	}
 }
 
+.el-switch.is-disabled .el-switch__core, .el-switch.is-disabled .el-switch__label {
+	cursor: auto !important;
+}
```

Comment:
This is to avoid having a `disabled` state cursor that block tooltip.

Reply:
This indeed affects all disabled switched (in case of node detail when prop is `readOnly`), is there a way to isolate this? Component styling is not sufficient

Reply:
we should be able to isolate it.. though in this case I don't think this change is needed